forge-gui/res/cardsfolder/upcoming/scavenged_brawler.txt

```diff
@@ -0,0 +1,11 @@
+Name:Scavenged Brawler
+ManaCost:6
+Types:Artifact Creature Construct
+PT:4/4
+K:Flying
+K:Vigilance
+K:Trample
+K:Lifelink
+A:AB$ PutCounter | Cost$ 5 ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard | ValidTgts$ Creature | SorcerySpeed$ True | CounterTypes$ P1P1,P1P1,P1P1,P1P1,Flying,Vigilance,Trample,Lifelink | SpellDescription$ Choose target creature. Put four +1/+1 counters, a flying counter, a vigilance counter, a trample counter, and a lifelink counter on that creature. Activate only as a sorcery.
```

Comment:
`Activate only as a sorcery` makes it into the stack description on this one, I bet... :(

I'm going to want to tweak out `CounterTypes` after this gets in so it can be something like `P1P1_4,Flying,...`
Maybe when I do that I can tweak the StackDesc builder for CounterTypes too